projects/ssr-tests/src/log.utils.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Contains methods pertaining to reading, writing and asserting of the ssr log
+ * generated by a running ssr server for the sake of testing ssr.
+ */
+
+import * as fs from 'fs';
+
+/**
+ * Path where SSR log file from server will be generated and read from.
+ */
+const SSR_LOG_PATH = './ssr.log';
+
+/**
+ * Writes no characters to log to clear log file.
+ */
+export function clearSsrLogFile(): void {
+  fs.writeFileSync(SSR_LOG_PATH, '');
+}
+
+/**
+ * Reads log and returns messages as string array.
+ */
+export function getLogMessages(): string[] {
+  const data = fs.readFileSync(SSR_LOG_PATH).toString();
+  return data
+    .toString()
+    .split('\n')
+    .filter((text: string) => text.charAt(0) === '{')
```

Comment:
It's not obvious why we filter lines starting with `{`, so IMHO it would be good to leave a code comment:

```suggestion
    // We're interested only in JSON logs from Spartacus SSR app.
    // We ignore plain text logs coming from other sources, like `Node Express server listening on http://localhost:4200`
    .filter((text: string) => text.charAt(0) === '{')
```